index.Rmd

```diff
@@ -6,10 +6,10 @@ subtitle: Presence/Absence of spotted turtle with wetlands dynamics
+Habitat fragmentation and degradation are undoubtedly a critcial player in the population dynamics of many species, due to humans manipulation of natural ecosystems. As the land cover on Earth is altered so drastically, we can witness species trying to react to such changes. This results in understanding how these species may or may not be able to react, resulting in conservation efferts. Understanding and assessing the dynamics of changes in wetlands and land cover provide evidence for species occurrence and where organisms will have suitable habitat to live. The spotted turtle is a small sized turtle with black coloration on the carapace, and bright yellow spots (Gibbs et al. 2007). They inhabit vernal pools in the spring, and upland forests in the summer, also inhabiting wet meadows, forested swamps, and sphagnum bogs (Gibbs et al. 2007). The spotted turtle can be found on the Great Lakes Plains south of Lake Ontario and east of Lake Erie, the lower Hudson river valley, and on Long Island. Aside from New York, spotted turtles are found as far north as Quebec, Canada, down to the state of Florida, and over to the central plains such as Illionis (Gibbs et al. 2007). The spotted turtle is declining throughout much of its range, and it is listed as a species of Special Concern in New York State. In New York the spotted turtle suffers from habitat fragmentation, wetland draining, suburban development, invasive wetland plants, road mortality (Lewis et al. 2004). Buchanan et al. 2017 states that their can be positive reactions from spotted turtles when specific parts of their habitat are specifically manipulated, however, Markle et al. 2018 warns that long-term habitat changes within specified habitats have implications for herpetofauna habitat management and restoration. With the assessment of wetlands in concert with occurrence citizen science data, the scope of this study is to address potential problems within habitat suitablity for the spotted turtle.       
 
```

Comment:
-Good detailed introduction but maybe can be divided up a bit into different section since it is well over the 200 word suggestion